using_listwheelscrollview/lib/roundcontainer.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+class NewWidget extends StatelessWidget {
+  final Widget l;
+  final String s;
+  double radius = 8;
+  NewWidget({this.size, this.l, this.s});
+  final double size;
+
+  @override
+  Widget build(BuildContext context) {
+    return AnimatedContainer(
+      duration: Duration(milliseconds: 1000),
+      curve: Curves.elasticInOut,
+      margin: EdgeInsets.symmetric(horizontal: 70, vertical: 20),
+      decoration: BoxDecoration(
+        color: Colors.black12,
+        borderRadius: BorderRadius.circular(20.0),
+        boxShadow: [
+          BoxShadow(
+            offset: Offset(5, 5),
+            blurRadius: radius,
+            color: Color(0XFF585858).withOpacity(.3),
+//            color: Color(0XFF383838).withOpacity(.4),

```

Comment:
Dead code?

Reply:
no sir, its a bit darker version for the button. kept as optional.

Reply:
Ok. Then if it is not used in the app, then lets remove this line of code.

Reply:
ok sir